tests/quickinspectortest.cpp

```diff
@@ -225,6 +226,93 @@ private slots:
         remoteView->setViewActive(false);
     }
 
+    void testPreviewFetchingThrottler()
+    {
+        auto remoteView
+            = ObjectBroker::object<RemoteViewInterface *>(QStringLiteral(
+                                                              "com.kdab.GammaRay.QuickRemoteView"));
+        QVERIFY(remoteView);
+
+        QVERIFY(showSource(QStringLiteral("qrc:/manual/rotationinvariant.qml")));
+
+        if (!isViewExposed())
+            return;
+
+        auto rootItem = view()->rootObject();
+        QVERIFY(rootItem);
+        Probe::instance()->selectObject(rootItem, QPoint());
+
+        connect(remoteView, SIGNAL(frameUpdated(GammaRay::RemoteViewFrame)), remoteView, SLOT(clientViewUpdated()));
+
+        // RemoteViewServer throttle the requests with an interval of ~16ms by default
+        qreal throttlerInterval = 100.0;
+        // Our animation properties
+        qreal animationInterval = 0.0;
+        qreal animationDuration = 1400.0;
```

Comment:
Testing for real frame timings looks extremely brittle, wouldn't this randomly break on high CI load?